controllers/token.go

```diff
@@ -30,8 +30,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-const (
-	defaultTokenTTL = 10 * time.Minute
+var (
```

Comment:
Should this still be a `const` rather than a `var`?

Reply:
Oh, that was a half-completed thought to make it configurable via the CLI. I've finished it, but I have no objection to tearing it back out and making it a `const` again.

Reply:
until we do make it configurable let's leave it as a const

Reply:
oh can you make this change @sethp-nr ?